### ISSUE TITLE
Bootstrap uses background images to convey meaning on some controls. This is considered a failure when testing for accessibility for people with disabilities.

### ISSUE THREAD

**Issue by mmolica2** [twbs/bootstrap#39446]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

Some components that use classes like form-select use a background image. form-select uses a downward carat background image to indicate that there is a dropdown to select from. Background images that convey meaning are considered a failure when testing for accessibility for individuals with disabilities. Using an image with an alt tag to convey meaning is generally considered acceptable.

### Motivation and context

This would make bootstrap more compliant with accessibility standards.

**Comment by patrickhlauke** [comment]
Closing as the general idea (for forced colour/WHCM) is captured here already https://github.com/twbs/bootstrap/issues/35941
